tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -1633,21 +1622,16 @@ bool CUDABlas::DoBlasGemm(
     }
   }
 
-  bool use_tensor_ops = false;
-#if CUDA_VERSION >= 9000
-  int cc_major, cc_minor;
-  stream->parent()->GetDeviceDescription().cuda_compute_capability(&cc_major,
-                                                                   &cc_minor);
-
-  // GPUs < sm_70 don't support tensor ops.
-  if (cc_major >= 7 && TensorOpMathEnabled()) {
-    use_tensor_ops = true;
+#if CUDA_VERSION < 11000
```

Comment:
When `9000 <=CUDA_VERSION < 11000`, we now set `CUBLAS_TENSOR_OP_MATH` even on devices with compute capability less than 7.0. On such devices, is `CUBLAS_TENSOR_OP_MATH` equivalent to `CUBLAS_DEFAULT_MATH`? If so, the check before was unnecessary.

Reply:
According to the documentation for cublasSetMathMode an architecture check was unnecessary.
```Note: The library falls back to the default math mode when Tensor Core operations are not supported or not permitted.```